superbit_lensing/medsmaker/superbit/medsmaker_mocks.py

```diff
@@ -386,9 +386,10 @@ def _select_sources_from_catalog(self, fullcat, catname='catalog.ldac', min_size
         # Choose sources based on quality cuts on this catalog.
         #keep = (self.catalog[size_key] > min_size) & (self.catalog[size_key] < max_size)
         #self.catalog = self.catalog[keep.nonzero()[0]]
-        self.logprint("Selecting analysis objects on CLASS_STAR...") # Adapt based on needs of data; FWHM~8 for empirical!
-        keep2 = self.catalog['CLASS_STAR']<0.92
-        self.catalog = self.catalog[keep2.nonzero()[0]]
+        self.logprint("Selecting analysis objects on FLUX_APER > 1 and SNR_WIN > 1...") # Adapt based on needs of data; FWHM~8 for empirical!
```

Comment:
Ah. I added this change when I was thinking about what to do with the `ncutout = 0` entries in the MEDS file, but after our discussion about the potential biases such a catalog-level cut would introduce, I didn't implement it. Instead, zero-cutout (noise) entires are removed from the MEDS file in the `filter_meds()` function. 

This coadd catalog-level cleaning is controlled by the `source_select` option in the top-level pipe config file, and is set to `False` by default. No runs in the last ~6 months have had any level of catalog cleaning!

I can imagine a few use cases where a catalog-level cleaning might be useful but it may be better to just remove the `_select_sources_from_catalog` function altogether.

Reply:
I agree with all this, but I'm going to keep the conversation up until the code has been removed so we don't forget